.github/workflows/continuous-deployment.yml

```diff
@@ -0,0 +1,77 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CD
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches: [ master ]
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    env:
+      IMAGE_NS: ibmcom
+      IMAGE_NAME: starter-kit-operator
+      IMAGE_TAG_PREFIX: 0.1.
+      IC_CR_NS: devex-ibmcom
+    
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      - name: Setup Go environment
+        uses: actions/setup-go@v2.1.2
+
+      - name: Install operator-sdk
+        env: 
+          SDK_RELEASE_VERSION: "v0.15.0"
+        run: |
+          curl -OJL https://github.com/operator-framework/operator-sdk/releases/download/${SDK_RELEASE_VERSION}/operator-sdk-${SDK_RELEASE_VERSION}-x86_64-linux-gnu
+          echo TODO we should verify the release binary https://docs.openshift.com/container-platform/4.1/applications/operator_sdk/osdk-getting-started.html
```

Comment:
The cluster we use is on OpenShift v4.3.31. Maybe we can use this version to be a bit more current?